openscenario/openscenario_interpreter/src/syntax/maneuver_group.cpp

```diff
@@ -35,6 +36,19 @@ ManeuverGroup::ManeuverGroup(const pugi::xml_node & node, Scope & scope)
   });
 }
 
+auto ManeuverGroup::run() -> void
+{
+  for (size_t index{0}; auto && maneuver : elements) {
```

Comment:
This repository is compiled with C++17. Don't take advantage of C++20 features yet.
```suggestion
  std::size_t index = 0;
  for (auto && maneuver : elements) {
```